src/app/static/src/tests/components/genericChart/plotly.test.ts

```diff
@@ -1,12 +1,19 @@
 // Mock the import of plotly so that we can spy on its exported 'newPlot' method - need to do this
 // before importing Plotly
 import Vuex from "vuex";
-
+import plotly from "plotly.js-basic-dist";
+const mocks = vi.hoisted(() => {
+    return {
+        newPlot: vi.fn(),
+        react: vi.fn()
+    }
+});
 vi.mock("plotly.js-basic-dist", () => ({
-    newPlot: vi.fn(),
-    react: vi.fn()
+    default: {
+        newPlot: mocks.newPlot,
+        react: mocks.react
+    }
```

Comment:
I'm a bit confused about what's going on here. `newPlot` and `react` are being mocked twice?

Reply:
yh i realised i can just do one of them, should be good now, removed the hoisted stuff